webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -1347,69 +1348,6 @@ def mocked_get(url, params, **options):
         ok_(dump['total'])
 
     @mock.patch('requests.get')
-    def test_carry_mware_error_codes(self, rget):
```

Comment:
You know, I think instead of deleting this test, we should replace it with something that makes sure that those same kinds of errors in the implementation will return appropriate errors to the client.